### PR TITLE
add Python Math.floor()

### LINKED ISSUE #4347: [Term Entry] Python Math Module floor()
### Reason/inspiration (optional)

"We would like a new term entry on the `math.floor()` function for [Python- Math Module](https://www.codecademy.com/resources/docs/python/math-module): `math.floor()`. The entry should go in a new file under `docs/content/python/concepts/math-module/terms/math-floor/math-floor.md.

The entry should include the following:
- A descrition of the function
- A `## Syntax` section that includes the syntax of the function and its details
- An `## Example` section that includes a ` ```python ``` ` codeblock and ` ```shell ``` ` that shows the example output.
- A `## Codebyte Example` that includes a ` ```codebyte/python ``` ` runnable codeblock

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by evankart**
### Description

Added a new term entry on the math.floor() function for [Python- Math Module](https://www.codecademy.com/resources/docs/python/math-module)

### Issue Solved

Closes #4347

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x ] All writings are my own.
- [ x] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/python/concepts/math-module/terms/math-floor/math-floor.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+The following example is runnable and uses the `math.floor()` function to return the floor of a number:
+
+```codebyte/python
+import math
+
+print(math.floor(4.2))
+# Output: 4
+
+print(math.floor(9.7))
+# Output: 9
```
Comment (ishg-153) [inline content/python/concepts/math-module/terms/math-floor/math-floor.md:48]:
```suggestion
import math

print(math.floor(4.2))

print(math.floor(9.7))
```
We want to get the users to run the codeblock and find the output themselves so no need to give the output.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@evankart Very nice and crisp entry. I have suggested a couple of changes. Please make those and let me know once done.

**Review by Helfese — APPROVED** [review]

**Review by helfese — APPROVED** [review]

File: content/python/concepts/math-module/terms/math-floor/math-floor.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import math
+
+print(math.floor(134.2))
+
+```
+
+The above code give the following output:
+
+```shell
+134
```
Comment (ishg-153) [inline content/python/concepts/math-module/terms/math-floor/math-floor.md:39]: Please add closing triple brackets
` ``` `
after these two lines

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@evankart one last change, everything else looks good.

File: content/python/concepts/math-module/terms/math-floor/math-floor.md (hunk 2 above)
Comment (evankart) [reply content/python/concepts/math-module/terms/math-floor/math-floor.md:39]: thanks @ishg-153, added now

**Review by ishg-153 — APPROVED** [review]
@evankart Looks good for a second review!

File: content/python/concepts/math-module/terms/math-floor/math-floor.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+```pseudo
+math.floor(n)
+```
+
+The **`math.floor()`** function takes in a value and returns its floor, the smallest integer greater than or equal to that value.
+
+## Example
+
+Use `math.floor()` to return the floor of `134.2`:
```
Comment (avdhoottt) [inline content/python/concepts/math-module/terms/math-floor/math-floor.md:27]:
```suggestion
Here's an example of `math.floor()` to return the floor of `134.2`:
```

File: content/python/concepts/math-module/terms/math-floor/math-floor.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+Takes in a numeric data type and rounds the value down to the nearest integer.
+
+## Syntax
+
+```pseudo
+math.floor(n)
+```
+
+The **`math.floor()`** function takes in a value and returns its floor, the smallest integer greater than or equal to that value.
```
Comment (avdhoottt) [inline content/python/concepts/math-module/terms/math-floor/math-floor.md:23]:
```suggestion
The `math.floor()` function takes in a value and returns its floor, the smallest integer greater than or equal to that value.
```

File: content/python/concepts/math-module/terms/math-floor/math-floor.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+Takes in a numeric data type and rounds the value down to the nearest integer.
```
Comment (avdhoottt) [inline content/python/concepts/math-module/terms/math-floor/math-floor.md:15]:
```suggestion
The **`math.floor()`** function takes in a numeric data type and rounds the value down to the nearest integer.
```

**Review by avdhoottt — APPROVED** [review]
LGTM!!

**Pull request merged**